packages/op-datasets/src/op_datasets/logic/transforms/daily_address_summary.py

```diff
@@ -0,0 +1,209 @@
+
+            -- Gas Price Breakdown, conversion for average calculation below
+            ,CAST(CAST((gas_price * receipt_gas_used) AS DECIMAL(38,9)) / constants.WEI_IN_GWEI AS DECIMAL(38,9)) AS total_gas_price_gwei
+            ,CAST(CAST(((gas_price - max_priority_fee_per_gas) * receipt_gas_used) AS DECIMAL(38,9)) / constants.WEI_IN_GWEI AS DECIMAL(38,9)) AS base_fee_gwei
+            ,CAST(CAST(max_priority_fee_per_gas * receipt_gas_used AS DECIMAL(38,9)) / constants.WEI_IN_GWEI AS DECIMAL(38,9)) AS priority_fee_gwei
+            ,CAST(CAST(receipt_l1_gas_price * receipt_l1_gas_used AS DECIMAL(38,9)) / constants.WEI_IN_GWEI AS DECIMAL(38,9)) AS l1_gas_price_gwei
+            ,CAST(CAST(receipt_l1_blob_base_fee * receipt_l1_gas_used AS DECIMAL(38,9)) / constants.WEI_IN_GWEI AS DECIMAL(38,9)) AS l1_blob_base_fee_gwei
+        FROM
+            _filter_df
+        CROSS JOIN constants
+    )
+
+    SELECT
+        {groupby_clause}
+
+        -- Transactions
+        ,COUNT(hash) AS total_txs
+        ,COUNT(CASE WHEN receipt_status = 1 THEN hash ELSE NULL END) AS total_txs_success
+        ,COUNT(CASE WHEN receipt_status != 1 THEN hash ELSE NULL END) AS total_txs_fail
+
+        -- Blocks
+        ,COUNT(DISTINCT block_number) AS total_blocks
+        ,COUNT(DISTINCT CASE WHEN receipt_status = 1 THEN block_number ELSE NULL END) AS total_blocks_success
+        ,COUNT(DISTINCT CASE WHEN receipt_status != 1 THEN block_number ELSE NULL END) AS total_blocks_fail
+        ,MIN(block_number) AS min_block_number
+        ,MAX(block_number) AS max_block_number
+        ,MAX(block_number) - MIN(block_number) + 1 AS block_interval_active
+
+        -- Nonce
+        ,MIN(nonce) AS min_nonce
+        ,MAX(nonce) AS max_nonce
+        ,MAX(nonce) - MIN(nonce) + 1 AS nonce_interval_active
+
+        -- Gas Usage
+        ,SUM(receipt_gas_used) AS total_l2_gas_used
+        ,SUM(CASE WHEN receipt_status = 1 THEN receipt_gas_used ELSE 0 END) AS total_l2_gas_used_success
+        ,SUM(CASE WHEN receipt_status != 1 THEN receipt_gas_used ELSE 0 END) AS total_l2_gas_used_fail
+        ,SUM(receipt_l1_gas_used) AS total_l1_gas_used
+        ,SUM(CASE WHEN receipt_status = 1 THEN receipt_l1_gas_used ELSE 0 END) AS total_l1_gas_used_success
+        ,SUM(CASE WHEN receipt_status != 1 THEN receipt_l1_gas_used ELSE 0 END) AS total_l1_gas_used_fail
+
+        -- Gas Fee Paid
+        ,SUM(total_gas_fee) AS total_gas_fees
+        ,SUM(total_gas_fee_success) AS total_gas_fees_success
+        ,SUM(total_gas_fee_fail) AS total_gas_fees_fail
+
+        -- Gas Fee Breakdown
+        ,SUM(l2_contrib_gas_fee) AS l2_contrib_gas_fees
+        ,SUM(l1_contrib_gas_fee) AS l1_contrib_gas_fees
+        ,SUM(l1_blobgas_contrib_gas_fee) AS l1_blobgas_contrib_gas_fees
+        ,SUM(l1_l1gas_contrib_gas_fee) AS l1_l1gas_contrib_gas_fees
+        ,SUM(l2_contrib_gas_fee_base_fee) AS l2_contrib_gas_fees_base_fees
```

Comment:
l1 and l2 metrics should have similar naming conventions? (i.e. l1_blobgas <> l2_contrib)

L2 has contrib, L1 doesn't